src/feat/wave-reader.cc

```diff
@@ -30,53 +30,59 @@
 
 namespace kaldi {
 
-// static
-void WaveData::Expect4ByteTag(std::istream &is, const char *expected) {
-  char tmp[5];
-  tmp[4] = '\0';
-  is.read(tmp, 4);
-  if (is.fail())
-    KALDI_ERR << "WaveData: expected " << expected << ", failed to read anything";
-  if (strcmp(tmp, expected))
-    KALDI_ERR << "WaveData: expected " << expected << ", got " << tmp;
-}
+// A utility class closure for reading wave header
```

Comment:
'closure' is not a concept that will be familiar to many readers; can you please rename this class and remove 'closure' from the comment?